mitre/system/ksp-mitre-t1222-002.yaml

```diff
@@ -0,0 +1,21 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-mitre-t1222-002
+  namespace: default # Change your namespace
+spec:
+  message: "Blocked the chmod and chown"
+  tags : ["MITRE","T1222.002"]
+  selector:
+    matchLabels:
+        app: ubuntu  # use your own label here
+  process:
+    matchPaths:
+    - path: /usr/bin/chmod
```

Comment:
What if binary present in other directory, like in my VM it's in residing in these location
```sh
/usr/bin/chmod
/bin/chmod
/bin//chmod
/bin/chmod
```